### PR TITLE
show status/signatures in contact-profiles

### PR DESCRIPTION
**Pull request by r10s**
- [x] add `DcContact.status` api
- [x] basically show status
- [x] make links clickable, maybe better use ~~`UILabel`~~ `MessageLabel` inside `MultilineLabelCell` for that purpose
- [x] make phone number clickable
- [x] make email addresses clickable
- [x] let `MultilineLabelCell` grow in height as needed

  

closes https://github.com/deltachat/deltachat-ios/issues/1055

### CODE REVIEW

**Comment by cyBerta** [comment]
>     * make links clickable, maybe better use `UILabel` inside `MultilineLabelCell` for that purpose

You could also use MessageLabel, it already has 'Detectors' to detect links, phone numbers etc.

**Comment by r10s** [comment]
> You could also use MessageLabel, it already has 'Detectors' to detect links, phone numbers etc.

sounds good!

File: deltachat-ios/ViewModel/ContactDetailViewModel.swift [hunk 1]
```diff
@@ -54,6 +56,12 @@ class ContactDetailViewModel {
         self.sharedChats = context.getChatlist(flags: 0, queryString: nil, queryId: contactId)
 
         sections.append(.chatOptions)
+
+        let dcContact = DcContact(id: contactId)
```
Comment (cyBerta) [inline deltachat-ios/ViewModel/ContactDetailViewModel.swift:60]: see line 32/33

File: deltachat-ios/Controller/ContactDetailViewController.swift [hunk 2]
```diff
@@ -430,3 +440,20 @@ class ContactDetailViewController: UITableViewController {
         }
     }
 }
+
+extension ContactDetailViewController: MultilineLabelCellDelegate {
+    func phoneNumberTapped(number: String) {
+        let sanitizedNumber = number.filter("0123456789".contains)
```
Comment (cyBerta) [inline deltachat-ios/Controller/ContactDetailViewController.swift:446]: what about '+' ? doesn't ios handle +1 for example correctly?

Comment (r10s) [reply deltachat-ios/Controller/ContactDetailViewController.swift:446]: i just copy+pasted from the existing code :)

**Comment by cyBerta** [comment]
Nice feature!
I have found one minor issue: 
rescaling of the fonts don't work immediatly. you have to leave the ViewController and reenter to get the correct scaling. 

see the attached image, the other cells do scale immediately which leads to such a screen:
Screen Shot 2021-05-26 at 14 52 31

**Comment by r10s** [comment]
> rescaling of the fonts don't work immediatly.

@cyBerta what is needed to get that correctly?

feel free to push sth. to this pr, maybe that is easier than explaining :)

**Review by cyBerta — APPROVED** [review]
the font issue will be targeted in another ticket

**Pull request merged**